src/theory/strings/skolem_cache.h

```diff
@@ -138,6 +138,8 @@ class SkolemCache
     //   where b is a regular expression, n is the number of occurrences of b
     //   in a, and k(0)=0.
     SK_OCCUR_LEN,
+    // exists a. 0 <= i < len(s) -> unit(a) = seq.at(s,i)
```

Comment:
Update comment to make clear that `a` is a function (analogous to the comments above).

Reply:
right! updated. Please let me know if any other changes are needed.